keps/sig-node/4639-oci-volume-source/kep.yaml

```diff
@@ -0,0 +1,54 @@
+title: OCI objects as VolumeSource
+kep-number: 4639
+authors:
+  - "@sallyom"
+  - "@saschagrunert"
+owning-sig: sig-node
+participating-sigs:
+  - sig-node
+  - sig-storage
+status: provisional
+creation-date: 2024-05-17
+reviewers:
+  - "@BigVan"
+  - "@alice.doe"
+  - "@aojea"
+  - "@humblec"
+  - "@mikebrow"
+  - "@mrunalp"
+  - "@rchincha"
+  - "@samuelkarp"
+  - "@sftim"
+  - "@syed"
+  - "@vsoch"
+  - "@xing-yang"
+approvers:
+  - "@sig-node-leads"
+  - "@sig-storage-leads"
+  - "@oscar.doe"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.31"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.31"
+  beta: "v1.32"
+  stable: "v1.33"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: OCIVolume
+    components:
+      - TBD
+disable-supported: true
+
+# The following PRR answers are required at beta release
+metrics:
+  - my_feature_metric
```

Comment:
(these aren't right; OK for a provisional KEP, but would need a fix before alpha)

Reply:
Removed for now :+1: